x-pack/plugins/reporting/server/browsers/chromium/paths.ts

```diff
@@ -7,32 +7,40 @@
 import path from 'path';
 
 export const paths = {
-  archivesPath: path.resolve(__dirname, '../../../.chromium'),
+  archivesPath: path.resolve(__dirname, '../../../../../../.chromium'),
```

Comment:
Could this use `fromRoot` ?

Reply:
`fromRoot` isn't currently exported to plugins.